tortoise/queryset.py

```diff
@@ -667,9 +667,7 @@ def _make_query(self) -> None:
                 if not isinstance(value, (F, ArithmeticExpression)):
```

Comment:
:thinking:  Should we change this to check for instance type `pypika.Term`? That is the common base class of both those?

Reply:
Looks okay. I changed it !